### DISCUSSION TITLE
my phrases disappear

### DISCUSSION THREAD

**Discussion by thracyturnix** [autokey/autokey#752] [Q&A]
I have tried a few times to enter phrases into autokey. Each time I "save" the phrases. 

When I reboot my pc, and open the main autokey window, The phrases are no longer there

I had a look in .config autokey folder and there are json files consistent with the phrase names I wrote

Can anyone advise why there are not loading / working in the main window when the pc starts?

https://i.imgur.com/rye4CYp.png

I guess I am missing a step somewhere but cant seem to figure out what

Help?

**Comment by josephj11** [comment]
What version of AutoKey are you using? It looks like you are using 0.95.10 or older. [0.96.0](https://github.com/autokey/autokey/releases/) is available in source and in deb packages with installation instructions [here](https://github.com/autokey/autokey/wiki/Installing).

It looks like you may have named your phrases with a leading dot (period). In Linux, such files are normally hidden from display. Don't do that. You may have uncovered a  UX bug in AutoKey. Ideally, it shouldn't allow you to do that, or at least warn you that it's not a good idea.

If I'm correct, your phrases are still there and might even work if you enter the trigger abbreviation or hotkey assigned to one of them in an appropriate window. They're just invisible!

if that's what happened, ignore the rest of this reply.

Rebooting shouldn't matter.

After you define a phrase, select Save from the main menu bar. The Save
button should change from enabled to disabled (grayed out) because there's
nothing further to save.

While still in the AutoKey window (without closing it), select another
folder such as Sample scripts and then select My Phrases again. Is your
phrase still there?

Optionally, Select Settings->Configure AutoKey->Automatically save changes
without confirmation. If you close the AutoKey window with this selected,
all your changes will be saved.

HTH

Joe

**Comment by josephj11** [comment]
I just recreated this. The "stealth" phrase works fine and is invisible in the AutoKey menu, but once AutoKey is terminated (not just closing the Menu window) and restarted, the phrase no longer works.

This is a known issue. See #325.

**Reply by thracyturnix** [reply]
thanks
